cypress/support/ui_commands.js

```diff
@@ -233,6 +233,17 @@ Cypress.Commands.add('getLastPostIdWithRetry', () => {
     return getLastPostIdWithRetry();
 });
 
+/**
+* Get post ID for nth newest post
+* .eq() is 0-based index, hence nthPost-2 to get the nth post
+@param {Integer} nthPost - nth newest post
+*/
+Cypress.Commands.add('getNthPostId', (nthPost) => {
+    return cy.get('#postListContent [id^=post]:first').parent().parent().siblings().eq(nthPost - 2).find('[id^=post]:first').invoke('attr', 'id').then((nthPostId) => {
+        return nthPostId.replace('post_', '');
+    });
+});
+
```

Comment:
💯

Reply:
I'm little confused on selector here, I got null when I tried to validate this on community.mattermost.com

@prapti Could you help me to understand this?

![image](https://user-images.githubusercontent.com/9696352/57176264-041f8a80-6e91-11e9-93c5-2f2410169f24.png)

Reply:
@cometkim This test was developed and verified against test-data and not necessarily against  community server. Is that a common approach for Cypress tests? But the general idea behind the step was to find out the older posts in the CENTER layout with a given post ID, similar to how the last post is identified.